.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
i think these are just your intellij settings so you probably shouldn't check in this file. i think you probably want to add ".idea/inspectionProfiles" to .gitignore. https://github.com/facebook-university/team_project/blob/master/.gitignore